subsquid/Makefile

```diff
@@ -44,5 +44,15 @@ up:
 down:
 	@docker-compose down
 
+rebuild:
+	rm -f -r lib
+	rm -f -r db/migrations
+	npx sqd db drop
```

Comment:
can we remove these, as they are deprecated `npx sqd`

Reply:
yes, well spotted